src/engine/server/sv_init.cpp

```diff
@@ -465,8 +465,9 @@ void SV_SpawnServer(const std::string pakname, const std::string server)
 		}
 	}
 
-	// allocate the snapshot entities on the hunk
-	svs.snapshotEntities = ( entityState_t * ) Hunk_Alloc( sizeof( entityState_t ) * svs.numSnapshotEntities, ha_pref::h_high );
+	// allocate the snapshot entities
+	free(svs.snapshotEntities);
+	svs.snapshotEntities = static_cast<entityState_t*>(calloc(svs.numSnapshotEntities, sizeof(entityState_t)));
```

Comment:
Please comment about the lifetime of the allocations.

Reply:
The lifetime was... it gets leaked when svs gets memset. I've changed it to use a unique_ptr and destroy svs instead of using memset.